lib/veri_helpers.mli

```diff
@@ -0,0 +1,5 @@
+open Core_kernel.Std
+open Bap.Std
+open Bap_traces.Std
+
+val bil_of_chunk: arch -> Chunk.t -> (string * bil) Or_error.t
```

Comment:
instead of `(string * bil)` use `insn` type. Also, it looks like that instead of `Or_error` it is better to use `Result` with a sum of possible errors.